src/consts/sub.ts

```diff
@@ -9,7 +10,7 @@ export const SUB_NETWORKS: Partial<Record<SubNetwork, NetworkData>> = {
     nativeCurrency: {
       name: 'KSM',
       symbol: 'KSM',
-      decimals: 18,
+      decimals: 12,
```

Comment:
where do these values come from?

Reply:
From chain properties, like `api.rpc.system.properties` - that return chain native token properties (symbol, decimals)